contracts/DelegatedStorage/DelegatedStorage.sol

```diff
@@ -0,0 +1,21 @@
+pragma solidity ^0.4.18;
+
+import '../Storage/BaseStorage.sol';
+import '../Proxy/DelegateManagerInterface.sol';
+import '../Proxy/DelegateManagable.sol';
+import './KeyManagerInterface.sol';
+
+contract DelegatedStorage is BaseStorage, DelegateManagable {
```

Comment:
thinking we should move everything that extends `BaseStorage` into the `/Storage` dir.